src/iop/highlights.c

```diff
@@ -1848,10 +1882,10 @@ static void process_visualize(dt_dev_pixelpipe_iop_t *piece, const void *const i
   const size_t width = roi_out->width;
   const size_t height = roi_out->height;
   const float clip = data->clip;
-  const float clips[4] = { clip * piece->pipe->dsc.temperature.coeffs[RED],
-                           clip * piece->pipe->dsc.temperature.coeffs[GREEN],
-                           clip * piece->pipe->dsc.temperature.coeffs[BLUE],
-                           clip * piece->pipe->dsc.temperature.coeffs[GREEN] };
+  const float clips[4] = { clip * fmaxf(1.0f, piece->pipe->dsc.temperature.coeffs[RED]),
```

Comment:
If coeff not defined we use 1.0f, fine. But if coeff is between 0 and 1.0f we are using 1.0f, looks suspicious to me, no? Is that true that coeff cannot be below 1.0f if defined?

Reply:
You mean by manually changing to "strange" coeffs in temperature?

Reply:
I think my question is more : what is are valid wb coeff range? Are these coeffs the values found in wb_presets.c (in the 4 channels) ?

Reply:
You worries are correct, all clips shoult only be set to 1.0 if undefined, otherwise are to be taken as_is.

See also the start of lab space correction ...

Reply:
Great! I'll test soon. Thanks.